app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/medication/MedicationDetailsFragment.kt

```diff
@@ -60,7 +63,10 @@ class MedicationDetailsFragment : Fragment(R.layout.fragment_medication_details)
     private fun setUpRecyclerView() {
         commentsAdapter = CommentsAdapter { parentEntryId ->
             val action = MedicationDetailsFragmentDirections
-                .actionMedicationDetailsFragmentToCommentsFragment(parentEntryId)
+                .actionMedicationDetailsFragmentToCommentsFragment(
+                    parentEntryId,
+                    userProfileId ?: ""
```

Comment:
userProfileId is hdId, which is available at repo level. Can we remove it from here?